x/minfee/go.mod

```diff
@@ -0,0 +1,136 @@
+module github.com/celestiaorg/celestia-app/x/minfee
```

Comment:
since we're already in v3 version in the app, shouldn't this also be v3?

Reply:
It could be but since there hasn't been a release of the Go module `github.com/celestiaorg/celestia-app/x/minfee` yet, I intended on creating v1.0.0

Reply:
Cool. If it won't create confusion for different modules having different versions used in a different celestia-app version, im fine with it :+1:

Reply:
TBH I do think it will be confusing. Because now the state machine modules will have versions different from the application.

But IMO it would make more sense for the `x/minfee` version to bump according to SemVer rather than arbitrarily bumping it alongside the app version. For example, we may cut celestia-app v4 soon with no changes to `x/minfee` and then the question is whether we re-release `x/minfee` v4.0.0

Reply:
Sounds good, thanks for clarifying